### PR TITLE
Add 'channels_fsm_count' to GetStatus report (changed: /debug/channels/fsm-count)

### PR DESCRIPTION
**Pull request by uwiger**
Adds `channels_fsm_count` to the HTTP `status` endpoint

Example:
```
GET "http://127.0.0.1:3013/v2/status"

*** Return code 200, Body
{"channel_fsms_count":2,
 "difficulty":1337,
 ...
}
```

### CODE REVIEW

File: apps/aehttp/test/aehttp_sc_SUITE.erl [hunk 1]
```diff
@@ -794,6 +798,10 @@ sc_ws_open_(Config, ChannelOpts0, MinBlocksToMine, LogDir) ->
             ok = ?WS:unregister_test_for_channel_events(IConnPid, TestEvents),
             ok = ?WS:unregister_test_for_channel_events(RConnPid, TestEvents)
     end,
+
+    CountAfter = get_channels_count(),
+    ct:log("CountAfter = ~p", [CountAfter]),
+
```
Comment (hanssv) [inline apps/aehttp/test/aehttp_sc_SUITE.erl:804]: Maybe check something w.r.t.`CountBefore` and `CountAfter`?

Comment (uwiger) [reply apps/aehttp/test/aehttp_sc_SUITE.erl:804]: I do that in the `aesc_fsm_SUITE`. For some reason, when I did that in the `aehttp_sc_SUITE`, the test run got stuck.

**Comment by uwiger** [comment]
Moved to a separate debug endpoint:

```
*** User 2022-10-27 16:57:12.408 ***
GET "http://127.0.0.1:3113/v2/debug/channels/fsm-count"

*** User 2022-10-27 16:57:12.410 ***
Return code 200, Body "{\"count\":0}"

*** User 2022-10-27 16:57:12.410 ***
CountBefore = 0
```

**Review by hanssv — APPROVED** [review]

File: apps/aehttp/priv/oas3.yaml [hunk 2]
```diff
@@ -1747,6 +1747,21 @@ paths:
...
+        - debug
+      operationId: GetChannelsFsmCount
+      description: Get the number of currently active State Channel FSMs
+      responses:
+        "200":
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/CountResponse"
```
Comment (velzevur) [inline apps/aehttp/priv/oas3.yaml:1764]: That is a neat reuse of the definition

**Review by velzevur — APPROVED** [review]

**Pull request merged**